sessionctx/variable/varsutil.go

```diff
@@ -488,9 +488,20 @@ func parseTimeZone(s string) (*time.Location, error) {
 	}
 
 	// The value can be given as a string indicating an offset from UTC, such as '+10:00' or '-6:00'.
+	// The time zone's value should in [-12:59,+13:00].
```

Comment:
I did some quick checking, and `+14:00` is possible: https://en.wikipedia.org/wiki/UTC%2B14:00

Reply:
Prior to Samoa introducing DST to its timezone in 2009, it probably didn't matter that MySQL only handled up to `+13:00`, since only [the line islands](https://en.wikipedia.org/wiki/Line_Islands) (6000 people) and [Chatham islands](https://en.wikipedia.org/wiki/Chatham_Islands) (600 people) were affected.  But Samoa is 195000 people. It is conceivable someone may need `+14:00` support, so we should break from MySQL here since the behavior is a bug.